pkg/operator/hive/hive_controller.go

```diff
@@ -92,6 +101,40 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Regular manager client is not fully initialized here, create our own for some
+	// initialization API communication:
+	tempClient, err := client.New(mgr.GetConfig(), client.Options{Scheme: mgr.GetScheme()})
+	if err != nil {
+		return err
+	}
+
+	// Determine if the openshift-config-managed namespace exists (> v4.0). If so, setup a watch
+	// for configmaps in that namespace.
+	ns := &corev1.Namespace{}
+	log.Debugf("checking for existence of the %s namespace", managedConfigNamespace)
+	err = tempClient.Get(context.TODO(), types.NamespacedName{Name: managedConfigNamespace}, ns)
+	if err != nil && !errors.IsNotFound(err) {
+		log.WithError(err).Errorf("error checking existence of the %s namespace", managedConfigNamespace)
+		return err
+	}
+	if err == nil {
+		log.Debugf("the %s namespace exists, setting up a watch for configmaps on it", managedConfigNamespace)
+		// Create an informer that only listens to events in the OpenShift managed namespace
+		kubeInformerFactory := kubeinformers.NewSharedInformerFactoryWithOptions(r.(*ReconcileHiveConfig).kubeClient, 10*time.Minute, kubeinformers.WithNamespace(managedConfigNamespace))
+		configMapInformer := kubeInformerFactory.Core().V1().ConfigMaps().Informer()
+		mgr.Add(&informerRunnable{informer: configMapInformer})
+
+		// Watch for changes to cm/kube-apiserver-aggregator-client-ca in the OpenShift managed namespace
+		err = c.Watch(&source.Informer{Informer: configMapInformer}, &handler.EnqueueRequestsFromMapFunc{ToRequests: handler.ToRequestsFunc(aggregatorCAConfigMapHandler)})
+		if err != nil {
+			return err
+		}
+		r.(*ReconcileHiveConfig).syncAggregatorCA = true
+		r.(*ReconcileHiveConfig).managedConfigCMLister = kubeInformerFactory.Core().V1().ConfigMaps().Lister()
+	} else {
+		log.Debugf("the %s namespace was not found, not watching for the aggregator CA configmap", managedConfigNamespace)
```

Comment:
I'd take this to Info and just a nit for brevity: "%s namespace not found, skipping watch for aggregator CR configmap".